### PR TITLE
Leslie: Week 1 homework pull request

### PR DESCRIPTION
**Pull request by noon-coffee**

### CODE REVIEW

File: routes/date.test.js [hunk 1]
```diff
@@ -4,7 +4,9 @@ const DateMock = require("jest-date-mock");
 const server = require("../server");
 
 beforeAll(() => {
-  DateMock.advanceTo(new Date(2020, 5, 26));
+  //fix: The "mock" month should be 4 since the months param is zero-based and the
```
Comment (t1mwillis) [inline routes/date.test.js:7]: I think you could also change the test to assert it returns 6 as well.

File: routes/parrot.test.js [hunk 2]
```diff
@@ -2,10 +2,12 @@ const request = require("supertest");
 
 const server = require("../server");
 
+//fix: in looking at the other tests, it appears as though the endpoint
+//needs to prefixed with the "base" route (don't know technical term)
 describe("/parrot", () => {
   describe("GET /parrot", () => {
     it("should return GET and query parameters", async () => {
-      const res = await request(server).get("/parrot?key1=true&key2=other");
+      const res = await request(server).get("/parrot/parrot?key1=true&key2=other");
```
Comment (t1mwillis) [inline routes/parrot.test.js:10]: Another way would be to remove the second `/parrot`

Comment (noon-coffee) [reply routes/parrot.test.js:10]: Ah, I think I understand and like your approach more - having a second "parrot" is redundant.

**Comment by noon-coffee** [comment]
Homework completed and graded.

**Pull request closed without merging**